Remotion/Core/UnitTests/Logging/Log4NetLogManagerTest.cs

```diff
@@ -185,10 +186,10 @@ public void InitializeConsole_WithSpecificThresholds_InvalidLog4NetLoggerType ()
       logger.Stub (stub => stub.Name).Return ("Foo");
       Assert.That (
           () => _logManager.InitializeConsole (LogLevel.Debug, new LogThreshold (new Log4NetLog (logger), LogLevel.Error)),
-          Throws.ArgumentException.With.Message.Matches (
+          Throws.ArgumentException.With.ArgumentExceptionMessageMatches (
```

Comment:
There are 2 tests where the message is checked with `matches`, so it should be no problem to undo the changes and only check the exception message

Reply:
```
.With.ArgumentExceptionMessageMatches (
                  "The object to be removed has the same ID as an object in this collection, but is a different object reference.",
                  "domainObject")
```
In this case the `Matches` check could also be an `EqualTo`, because the whole message must be equal and not just a specific pattern.